beacon-chain/sync/validate_blob.go

```diff
@@ -169,8 +169,8 @@ func blobFields(b blocks.ROBlob) logrus.Fields {
 	}
 }
 
-func computeSubnetForBlobSidecar(index uint64) uint64 {
-	return index % params.BeaconConfig().BlobsidecarSubnetCount
+func computeSubnetForBlobSidecar(index uint64, slot primitives.Slot) uint64 {
+	return index % uint64(params.BeaconConfig().MaxBlobsPerBlock(slot))
```

Comment:
I see a `params.BeaconConfig().BlobsidecarSubnetCountElectra` why isn't that used instead?

Reply:
I thought checking against max blobs per block is cleaner but you are right I should use that instead: https://github.com/prysmaticlabs/prysm/pull/14750/commits/4e62a244cd992e5c3e118c472b4ee43c3b21074b